cmd/rpcdaemon/commands/eth_api.go

```diff
@@ -464,8 +466,12 @@ func newRPCTransaction(tx types.Transaction, blockHash common.Hash, blockNumber
 		if t.IsSystemTransaction {
 			result.IsSystemTx = &t.IsSystemTransaction
 		}
-		if depositNonce != nil {
-			result.Nonce = hexutil.Uint64(*depositNonce)
+		if receipt != nil && receipt.DepositNonce != nil {
+			result.Nonce = hexutil.Uint64(*receipt.DepositNonce)
+			if receipt.DepositReceiptVersion != nil {
+				result.DepositReceiptVersion = new(hexutil.Uint64)
+				*result.DepositReceiptVersion = hexutil.Uint64(*receipt.DepositReceiptVersion)
```

Comment:
```suggestion
				depositReceiptVersion := *receipt.DepositReceiptVersion
				result.DepositReceiptVersion = &depositReceiptVersion
```

Reply:
This is a style preference, and I would like to follow what op-geth did, for easier tracking.